pkgs/applications/networking/instant-messengers/signal-desktop/default.nix

```diff
@@ -51,61 +69,46 @@ let
     libXrandr
     libXrender
     libXtst
+    libappindicator-gtk3
+    libnotify
+    libuuid
     nspr
     nss
-    udev
+    pango
+    systemd
     xorg.libxcb
   ];
 
-in stdenv.mkDerivation rec {
-  pname = "signal-desktop";
-  version = "1.29.6"; # Please backport all updates to the stable channel.
-  # All releases have a limited lifetime and "expire" 90 days after the release.
-  # When releases "expire" the application becomes unusable until an update is
-  # applied. The expiration date for the current release can be extracted with:
-  # $ grep -a "^{\"buildExpiration" "${signal-desktop}/libexec/resources/app.asar"
-  # (Alternatively we could try to patch the asar archive, but that requires a
-  # few additional steps and might not be the best idea.)
-
-  src = fetchurl {
-    url = "https://updates.signal.org/desktop/apt/pool/main/s/signal-desktop/signal-desktop_${version}_amd64.deb";
-    sha256 = "1s1rc4kyv0nxz5fy5ia7fflphf3izk80ks71q4wd67k1g9lvcw24";
-  };
-
-  phases = [ "unpackPhase" "installPhase" ];
-
-  nativeBuildInputs = [ dpkg wrapGAppsHook ];
+  runtimeDependencies = [
+    systemd.lib
+  ];
 
   unpackPhase = "dpkg-deb -x $src .";
 
+  dontBuild = true;
+  dontConfigure = true;
+
   installPhase = ''
-    mkdir -p $out
-    cp -R opt $out
+    mkdir -p $out/lib
```

Comment:
Optional: Do we really need `/lib` here?

Reply:
Yes, else `$out/lib` won't have a `Signal` subdir.

Reply:
Oh, sorry, I was somehow completely confused why the `mv` command didn't create a subdir as expected, probably missed that the mv src dir was called `Signal` and not `lib` and was therefore expecting `lib/lib`...